Purchasing.Mvc/Controllers/OrderController.cs

```diff
@@ -1201,10 +1203,9 @@ public ActionResult AddVendor(int workgroupId, WorkgroupVendor vendor)
 
             Check.Require(_securityService.HasWorkgroupAccess(workgroup));
 
-            var modelState = new ModelStateDictionary();
```

Comment:
@sprucely I think this changes the functionality and should probably be reverted.